### ISSUE TITLE
Directories marked as excluded are reported as never used

### ISSUE THREAD

**Issue by stof** [hsz/idea-gitignore#571]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [ ] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Steps to Reproduce

1. Create a `web/build/` folder in a project
2. Add `/web/build/` in your `.gitignore`
3. Mark the `web/build/` as excluded in PHPStorm's setting (can be done from the context menu or from the `Directories` panel of the settings)
4. Open the `.gitignore` file

**Expected behavior:**

The `/web/build/` rule is detected as being used

**Actual behavior:** 

The `/web/build/` rule is reported as being unused

**Reproduces how often:** 100%

### Versions

**Plugin:** 

```
v3.0.0.182
```

**IDE:** 

```
PhpStorm 2018.3.2
Build #PS-183.4886.46, built on December 19, 2018

JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.4.0-141-generic
```

**OS:** 

```
Ubuntu 16.04
```

**Comment by foobar13372** [comment]
Can confirm.

**Comment by hsz** [comment]
@stof @foobar13372 
Unfortunatelly this cannot be fully fixed - I'll explain in a moment, why.

Rules in the `.gitignore` file are analyzed using the Project scope's files (simplifying - project view). If you're excluding something from the project/IDE - it's not available anymore for the PsiManager - I cannot find the indexed file's reference.

The only thing that can be achieved is to - at the very end of some checks - reveal if the ignore rule refers to the exisiting file in the file system. If so, rule will not be marked as _unused_.
Downside of this solution is that this rule will not navigate you to the proper file (as said, no PsiReference available).

I hope, that it'll be enough for this kind of problem.

Solution will be delivered with upcoming release v3.1.1.

**Comment by hsz** [comment]
Fixed and released with v3.1.1
